guides/common/modules/proc_creating-custom-rhel-images.adoc

```diff
@@ -5,10 +5,8 @@
 * Use a Linux host machine to create an image.
 In this example, we use a {RHEL} 7 Workstation.
 * Use `virt-manager` on your workstation to complete this procedure.
-If you create the image on a remote server, connect to the server from your workstation with `virt-manager`.
```

Comment:
This file is about clients so I'd suggest to revert all changes.

Reply:
Addressed :+1: